unlock-app/src/components/interface/checkout/main/Confirm/ConfirmCrossChainPurchase.tsx

```diff
@@ -135,14 +153,26 @@ export function ConfirmCrossChainPurchase({
       delete route.tx.maxFeePerGas
       delete route.tx.maxPriorityFeePerGas
 
-      const tx = await walletService.signer.sendTransaction(route.tx)
-      onConfirmed(lockAddress, route.network, tx.hash)
+      let tx
+      // if embedded wallet, we need to use the sendTransaction method from Privy
```

Comment:
This probably needs to be extracted because it will also be useful for `ConfirmCrypto`